packages/uswds-core/src/js/polyfills/svg4everybody.js

```diff
@@ -30,7 +30,7 @@
       if (use) {
         for (var i = 0; use.attributes.length > i; i++) {
           var attr = use.attributes[i];
-          "xlink:href" !== attr.name &&
+          "href" !== attr.name &&
             "href" !== attr.name &&
```

Comment:
**polish:** Since this is checking for `xlink:href && href` here we can just remove this entire line.

There also appears to be another `xlink:href` reference on line 96 of this file:

https://github.com/uswds/uswds/blob/4f73861619bf0337f503ac4449cac54ca42097f0/packages/uswds-core/src/js/polyfills/svg4everybody.js#L96

Not sure if this needs to be a blocker since we're removing this entire file in another PR but might be good for cleanliness